core/api/v1/keytransparency.proto

```diff
@@ -338,6 +338,15 @@ message MapRoot {
   repeated bytes log_inclusion = 2;
 }
 
+// LogRootRequest contains the information needed to requetst and verify LogRoot.
+message LogRootRequest {
+  // root_hash is the log root hash.
+  bytes root_hash = 1;
```

Comment:
Why does the log root request contain the log root hash? Isn't it supposed to be returned as a result of the request?

Reply:
The `LogRootRequest` is intended to encode the client's verifier state, as is being done in this PR; but it has a second purpose, which is to make request, response pairs statelessly verifiable.  At the moment, an observer of the system, or a server interceptor can't tell if a response is valid because the first hash of the consistency proof is not included in either the request or the response.